packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -258,14 +261,13 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface, Ownable {
         return SovereignSecurityManagerInterface(assertions[assertionId].sovereignSecurityManager);
     }
 
-    function _getAssertionPolicies(bytes32 assertionId)
```

Comment:
curious on why you re-named this. in thinking about this perhaps the most descriptive name could be `_getAssertionPolicyFromSsm` wdyt?

Reply:
the same reason why renamed `getAssertionPolicies` -> `processAssertionPolicies` in SSM since this method now can change state on the SSM side (e.g. update claim bonding trackers)